litmus-portal/frontend/src/components/Sections/Workflow/ChooseWorkflow/index.tsx

```diff
@@ -102,12 +103,17 @@ const ChooseWorkflow: React.FC = () => {
       id: workflowsList[index].workflowID,
       yaml: 'none',
       description: workflowsList[index].description,
-      isCustomWorkflow: false,
+      isCustomWorkflow: workflowsList[index].isCustom,
```

Comment:
This change is okay however we are not having the custom workflow card in this release so the value would always be `false`.

Also the data above needs to scope out of this component since it is a part of pre-defined workflow, not a good idea to scope it only to this section.

Reply:
But pre-defined workflow are getting used in this component.